src/index.ts

```diff
@@ -31,7 +31,7 @@ export class DatastoreStore extends expressSession.Store {
   kind: string;
   expirationMs: number;
   constructor(options: DataStoreOptions = {}) {
-    super(options);
```

Comment:
Should I not be worried about that?

Reply:
Yes, but tests pass ... and I looked at the codebase of `express-session` and it doesn't use these options as far as I can tell.